apps/mobile/mobile-app/components/ui/utils/use-break-point-value.ts

```
@@ -1,103 +1,103 @@
+
+		mediaQueriesBreakpoints.push({
+			key: key,
+			breakpoint: resolveScreenWidth[key],
+			isValid: isValid,
+		})
+	})
+
+	mediaQueriesBreakpoints.sort((a: any, b: any) => a.breakpoint - b.breakpoint)
+
+	mediaQueriesBreakpoints.forEach((breakpoint: any, index: any) => {
+		breakpoint.value = values.hasOwnProperty(breakpoint.key)
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)